src/sickle.ts

```diff
@@ -15,6 +15,16 @@ export interface SickleOutput {
   diagnostics: ts.Diagnostic[];
 }
 
+/** The compiler options used by the test suite and sickle command line. */
+export const compilerOptions: ts.CompilerOptions = {
+  target: ts.ScriptTarget.ES6,
+  noImplicitAny: true,
+  skipDefaultLibCheck: true,
```

Comment:
From the docs: "Do not add triple-slash references or module import targets to the list of compiled files."

This took me like an hour to figure out why I couldn't get the damn test to load the extra module... :~(